src/command.rs

```diff
@@ -27,6 +27,31 @@ pub struct Command<'a> {
     pub handler: Box<Handler<'a>>,
 }
 
+/// Command equals when types equal
```

Comment:
I feel like implementing equality for `Command` might be a bit confusing. We would be using only the types of args, so arg names, description and handler are ignored. `Command` is `pub`, so maybe it's better to avoid Eq. We could provide a method `fn arg_types(&self) -> Vec<String>` and we would then compare `cmd1.arg_types() == cmd2.arg_types()`.